docs/Q-and-A.md

```diff
@@ -254,6 +262,36 @@ No, including. And it doesn't really require that much, because there aren't rea
 https://matthijs.hoekstraonline.net/2021/03/25/running-an-ion-node-on-your-raspberry-4/
 
 
+## *Q: What are the availability guarantees of ION?
+ION does not depend on our infra in a way that somehow fails if our node is down. All ION nodes aggressively replicate all operations.\
+(_@csuwildcat_)
```

Comment:
I don't think the answer should center around "out (Microsoft's)" node. I'd say the answer should  be along the lines of:

It depends on the aggregated availability of all the nodes. The more ION nodes there are, the more available and redundant ION data is.

Reply:
Thx, Henry, all your suggestions committed. I've changed the line above to:
ION depends on the aggregated availability of all the nodes. The more ION nodes there are, the more available and redundant ION data is. ION does not depend on single infra in a way that it somehow fails if a few nodes are down. All ION nodes aggressively replicate all operations.\
(_@csuwildcat_ and _@thehenrytsai_)